js/src/Map.js

```diff
@@ -33,8 +36,8 @@ export class LeafletMapModel extends widgets.DOMWidgetModel {
       _model_name: 'LeafletMapModel',
       _model_module: 'jupyter-leaflet',
       _view_module: 'jupyter-leaflet',
-
       center: DEFAULT_LOCATION,
+      continuousWorld: false,
```

Comment:
Why did you add this?

If you want to add a new option, it must be written in `snake_case`

Reply:
Also, we might want to make it configurable from the python side. Some projections may need the tile coordinates to be wrapped?
I can see that this option [used to exist](https://leafletjs.com/reference-0.7.7.html#tilelayer-continuousworld) but [doesn't seem to be present anymore](https://leafletjs.com/reference-1.0.3.html) in Leaflet 1.0.3, which Proj4Leaflet 1.0.1 uses.

Reply:
Then maybe adding it would not be a good idea? If we need to remove it soon.

Reply:
I included it to experiment with my polar projections but had no effect on what I was trying to achieve at the moment. Maybe removing it altogether is the better option here.

Reply:
Agreed. But it could be added in a separate.

Reply:
I would be good to understand what it is for. The [web site](https://kartena.github.io/Proj4Leaflet) says: "note the use of the option continuousWorld for the tile layer, which tells Leaflet that the tiles should not be wrapped the same way they should for its default projection".

Reply:
But it's definitely not there anymore (https://github.com/Leaflet/Leaflet/pull/4058), so yes we should just remove it.

Reply:
done!